Library/Homebrew/tab.rb

```diff
@@ -28,7 +28,7 @@ def self.create(formula, compiler, stdlib, build)
       "stdlib" => stdlib,
       "source" => {
         "path" => formula.path.to_s,
-        "tap" => formula.tap,
+        "tap" => formula.tap.is_a?(Tap) ? formula.tap.name : formula.tap,
```

Comment:
This isn't obvious for a `nil` check; feels like it should be just `formula.tap ? formula.tap.name : nil` if that's what you're doing.